lib/rules/ruleWarningMessages.js

```diff
@@ -0,0 +1,23 @@
+import entries from 'object.entries';
```

Comment:
I use node version 6.8.1. But if this would be a problem, then we will never pass the node -v 4 travis test.